model/src/main/kotlin/licenses/LicenseConfiguration.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.licenses
+
+import com.fasterxml.jackson.annotation.JsonInclude
+
+/**
+ * A configuration for licenses which allows to assign meta data to licenses. It allows assigning licenses to sets which
+ * can be useful for making license classifications. The available licenses sets are implicitly defined by the
```

Comment:
* Again, this is implement specifically for license classifications, so I would say the "are useful".
* "...license**~~s~~** sets..."